webapp/tests/karma/ts/modules/messages/messages-content.component.spec.ts

```diff
@@ -150,7 +150,7 @@ describe('MessagesContentComponent', () => {
       expect(performanceService.track.calledOnce).to.be.true;
       expect(stopPerformanceTrackStub.calledOnce).to.be.true;
       expect(stopPerformanceTrackStub.args[0][0]).to.deep.equal({
-        name: 'select_conversation:load',
+        name: 'messages_detail:load',
```

Comment:
Do we keep this as plural or make it singular?  The other entries are singular i.e `report_detail`, `contact_detail`

Reply:
Plural, see conversation [here](https://medic.slack.com/archives/C02GME3BQE6/p1707191934904099?thread_ts=1707166546.743469&cid=C02GME3BQE6). Because when clicking on one item from the list, it loads a conversation containing many messages. 
Different from reports that load 1 report, for example.